### PR TITLE
[Term Entry] Kotlin Strings .lowercase()

### PR DESCRIPTION
**Pull request by Fejiro001**
### Description

- Added the .lowercase() method for Kotlin Strings
- Closes Issue #3980    

### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/kotlin/concepts/strings/terms/lowercase/lowercase.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+Title: '.lowercase()'
+Description: 'Compares two strings lexicographically.'
```
Comment (dakshdeepHERE) [inline content/kotlin/concepts/strings/terms/lowercase/lowercase.md:3]:
```suggestion
Description: 'It is used to convert a string to lowercase'
```

File: content/kotlin/concepts/strings/terms/lowercase/lowercase.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  - 'learn-kotlin'
+  - 'paths/computer-science'
+---
+
+The **`.lowercase()`** method in Kotlin is changes all characters in a string to lowercase. It does not modify the original string, rather it returns a new transformed string. This is useful when case-sensitive operations are performed.
+
+## Syntax
+
+```pseudo
+String.lowercase(): String
```
Comment (dakshdeepHERE) [inline content/kotlin/concepts/strings/terms/lowercase/lowercase.md:20]:
```suggestion
String.lowercase()
```
I don't think we have to mention the return type it's very obvious.

File: content/kotlin/concepts/strings/terms/lowercase/lowercase.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+This example shows how to use the `.lowercase()` method on a string:
+
+```kotlin
+fun main() {
+    val originalString = "Hello, World!"
+    val newString = originalString.lowercase()
+    println("Original: $originalString")
+    println("Lowercase: $newString")
+}
```
Comment (dakshdeepHERE) [inline content/kotlin/concepts/strings/terms/lowercase/lowercase.md:35]:
```suggestion
fun main() {
  val originalString = "Hello, World!"
  val newString = originalString.lowercase()
  println("Original String: $originalString")
  println("Lowercased String: $newString")
}
```

File: content/kotlin/concepts/strings/terms/lowercase/lowercase.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+```
+
+The code will return the following output:
```
Comment (dakshdeepHERE) [inline content/kotlin/concepts/strings/terms/lowercase/lowercase.md:38]:
```suggestion
The above code will return the following output:
```

File: content/kotlin/concepts/strings/terms/lowercase/lowercase.md [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+
+```shell
+Original: Hello, World!
+Lowercase: hello, world!
```
Comment (dakshdeepHERE) [inline content/kotlin/concepts/strings/terms/lowercase/lowercase.md:42]:
```suggestion
Original String: Hello, World!
Lowercased String: hello, world!
```

**Review by dakshdeepHERE — CHANGES_REQUESTED** [review]
I have few comments from my side for you to work on @Fejiro001 🚀

**Review by dakshdeepHERE — APPROVED** [review]
Hey @Fejiro001, thank you for committing the changes. This looks good to me, let's move it to the second review.

File: content/kotlin/concepts/strings/terms/lowercase/lowercase.md [hunk 6]
```diff
@@ -0,0 +1,43 @@
+---
+Title: '.lowercase()'
+Description: 'It used to convert a string to lowercase.'
```
Comment (avdhoottt) [inline content/kotlin/concepts/strings/terms/lowercase/lowercase.md:3]:
```suggestion
Description: 'Converts a string to lowercase.'
```

File: content/kotlin/concepts/strings/terms/lowercase/lowercase.md [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+  - 'Code Foundations'
+Tags:
+  - 'Strings'
+  - 'Methods'
+CatalogContent:
+  - 'learn-kotlin'
+  - 'paths/computer-science'
+---
+
+The **`.lowercase()`** method in Kotlin is changes all characters in a string to lowercase. It does not modify the original string, rather it returns a new transformed string. This is useful when case-sensitive operations are performed.
```
Comment (avdhoottt) [inline content/kotlin/concepts/strings/terms/lowercase/lowercase.md:15]:
```suggestion
The **`.lowercase()`** method in Kotlin changes all characters in a string to lowercase. It does not modify the original string, rather it returns a new transformed string. This is useful when case-sensitive operations are performed.
```

File: content/kotlin/concepts/strings/terms/lowercase/lowercase.md [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+
+```pseudo
+String.lowercase()
+```
+
+- `String`: The string to be converted to lowercase.
+
+## Example
+
+This example shows how to use the `.lowercase()` method on a string:
```
Comment (avdhoottt) [inline content/kotlin/concepts/strings/terms/lowercase/lowercase.md:27]:
```suggestion
The following example shows how to use the `.lowercase()` method on a string:
```

**Review by avdhoottt — CHANGES_REQUESTED** [review]
@Fejiro001 Mentioned a few changes, please make them.

**Comment by avdhoottt** [comment]
@Fejiro001 It's been a week since I suggested the changes. Are you working on this PR?

**Review by avdhoottt — APPROVED** [review]
Looks good to get merged!!

**Pull request merged**